src/server/create/views/disabled.njk

```diff
@@ -0,0 +1,33 @@
+{% extends "layouts/page.njk" %}
+
+
+{% block content %}
+
+  <div class="govuk-main-wrapper app-main-wrapper">
+
+    {{ appHeading({
+      title: heading,
+      caption: headingCaption
+    }) }}
+
+    <div class="govuk-grid-row">
+      <div class="app-grid-column app-grid-column-one-half-from-desktop-wide">
+
+        <h2 class="govuk-heading-l govuk-!-margin-bottom-2">Temporarily disabled</h2>
```

Comment:
https://design-system.service.gov.uk/patterns/service-unavailable-pages/
Some GDS guidelines on these sort of pages.

I'd lean towards something like 'undergoing maintenance'  rather than disabled. 

I'd probably remove the 'try again in an hour' bit, it sort of contradicts the line above saying it will be available very soon. instead just direct them to the support channel for updates

Reply:
Yeah makes sense. Ill snazz it up to gds standard as well